config/lib/express.js

```diff
@@ -89,6 +90,21 @@ module.exports.initMiddleware = function (app) {
   // Add the cookie parser and flash middleware
   app.use(cookieParser());
   app.use(flash());
+
+  // Authorize JWT
+  app.use(function (req, res, next) {
```

Comment:
I think it would be better to use the new `config/lib/authorization` service as the middleware here.

Something like:
```javascript
app.use(authorization.authorize);
```

Any objections? Or other input on this part of the implementation?